src/models/ClassPotentialCPintheDark.cpp

```diff
@@ -1248,10 +1248,6 @@ void Class_Potential_CPintheDark::write() const
  */
 std::vector<double> Class_Potential_CPintheDark::calc_CT() const
 {
-
-  typedef std::numeric_limits<double> dbl;
-  std::setprecision(dbl::max_digits10);
```

Comment:
is this a global/consistent choice?

Reply:
This actually did nothing, because you need to pipe the return to a stream, e.g. os << std::setprecision(....). 
To preserve the current behaviour I just removed those unused lines of code.